src/shared/foundation-shared-components/components/FSDialogMultiFormBody.vue

```diff
@@ -3,10 +3,48 @@
     gap="24px"
   >
     <FSPagination
+      v-if="$props.mode === 'pagination'"
       width="calc(100% - 16px)"
       :pages="$props.steps"
       :modelValue="currentStep - 1"
     />
+    <FSTabs
+      v-else-if="$props.mode === 'tabs'"
+      :tab="currentStep - 1"
+      :color="$props.tabsColor"
+      @update:tab="(val) => currentStep = val + 1"
+    >
+      <FSTab
+        v-for="(step, index) in $props.steps"
+        :key="index"
+      >
+        <template
+          v-if="$slots[`tab-${step}-icon`] "
+          #prepend
+        >
+          <FSIcon
+            size="m"
+          >
+            <slot
+              :name="`tab-${step}-icon`"
+            />
+          </FSIcon>
+        </template>
+        <template
+          #label
+        >
+          <FSSpan
+            font="text-button"
+          >
+            <slot
+              :name="`tab-${step}-label`"
+            >
+              Step {{ step }}
```

Comment:
Mettre une traduction avec argument type : 

```suggestion
              {{ $tr('ui.tabs.step-number', 'Step {0}', step) }}
```